protos/sigstore_trustroot.proto

```diff
@@ -67,15 +68,23 @@ message CertificateAuthority {
 // to capture the complete/global set of trusted verification materials.
 // When verifying an artifact, based on the artifact and policies, a selection
 // of keys/authorities are expected to be extracted and provided to the
-// verification function. This way the set of keys/authorities kan be kept to
+// verification function. This way the set of keys/authorities can be kept to
 // a minimal set by the policy to gain better control over what signatures
 // that are allowed.
+//
+// The embedded transparency logs, CT logs, CAs and TSAs MUST include any
+// previously used instance -- otherwise signatures made in the pas cannot
+// be verified.
+// The currently used instances MUST NOT have their 'end' timestamp set in
+// their 'valid_for' attribute for easy identification.
+// All the listed instances SHOULD be sorted by the 'valid_for' in ascending
+// order, that is, the oldest instance first and the current instance last.
```

Comment:
Just curious why this matters?

Reply:
From programmatic perspective, it doesn't. But it's always nice to have items ordered when they can, and it makes it easier to inspect/work with the files for humans.